pkgs/applications/networking/mailreaders/neomutt/default.nix

```diff
@@ -86,10 +83,8 @@ stdenv.mkDerivation rec {
     (cd test-files && ./setup.sh)
 
     export NEOMUTT_TEST_DIR=$(pwd)/test-files
-    export LC_ALL="en_US.UTF-8"
```

Comment:
Not sure why this was required however at leas on you Linux you can use `C.utf-8`